skr/webhook/chart/skr-webhook/templates/deployment.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: v1
+kind: Service
+metadata:
```

Comment:
I think all these templates should be created via kustomize if we already have it as a dependency, that way we do not have to introduce 2 systems.

Reply:
https://github.com/kyma-project/kyma-watcher/issues/42